resizeobserver/demo.js

```diff
@@ -0,0 +1,13 @@
+/* global ResizeObserver */
+
+const ro = new ResizeObserver(entries => {
```

Comment:
Ah! I now I understand. Thanks for clarifying. The continuous bit is indeed good to show off. Will change :)